components/Conversation/MessageComposer.tsx

```diff
@@ -58,12 +59,11 @@ const MessageComposer = ({ onSend }: MessageComposerProps): JSX.Element => {
           required
         />
         <button type="submit" className={messageComposerStyles.arrow}>
-          <img
-            src={message ? upArrowGreen : upArrowGrey}
-            alt="send"
-            height={32}
-            width={32}
-          />
+          {!message ? (
+            <Image src={upArrowGrey} alt="send" height={32} width={32} />

```

Comment:
What advantages does the switch to Next Images give us? Are there any drawbacks?

Reply:
Image component uses optimization for images that's the only advantage, we can configure other things like cache time etc for these images also, but mainly to reduce the size of the image before serving is the reason to use this